src/modules/pages/EditPage/ApiItem.js

```diff
@@ -41,8 +41,10 @@ class ApiItem extends PureComponent {
         getUpdatedEndpoint(values),
     )(this.props.selectedPlugins);
 
-    handleDelete = (apiName) => {
-        this.props.deleteEndpoint(apiName);
+    handleDelete = () => {
+        const needRedirect = true;
```

Comment:
why not just deleteEndpoint(this.props.api, true);?

Reply:
just for more explicit showing what exactly this `true` is doing.